Yet again, we're not using the commandSubscriber and using the eventSubscriber instead

internal/platform/container/container.go:46:2: `commandSubscriber` is unused (structcheck)
        commandSubscriber message.Subscriber
        ^
make: *** [lint] Error 1

--- a/internal/platform/container/container.go
+++ b/internal/platform/container/container.go
@@ -243,7 +243,7 @@ func (c *container) commandPublisher() message.Publisher {
 
 func (c *container) commandSubscriber() message.Subscriber {
 	if c.eventSourcing.commandSubscriber != nil {
-		return c.eventSourcing.eventSubscriber
+		return c.eventSourcing.commandSubscriber
 	}
 	c.eventSourcing.commandSubscriber = c.eventSourcing.eventDriver.commandSubscriber()
 	return c.eventSourcing.commandSubscriber

